roles/openshift_logging/tasks/install_logging.yaml

```diff
@@ -51,6 +51,26 @@
     loop_var: file
   when: ansible_check_mode
 
+- name: Link Pull Secrets With Service Accounts
```

Comment:
It's pretty inconvenient that we aren't creating service accounts with a module meaning we need to add a pull secret out here...  can you add a TODO comment for this just as a reminder so we don't lose track of that? I'd like to make sure that when we have a module to create a service account idempotently that we would create a pull secret shortly after if applicable.